ts/src/time-profiler-bindings.ts

```diff
@@ -28,8 +28,9 @@ export function startProfiling(runName: string, includeLineInfo?: boolean) {
 }
 
 export function stopProfiling(
-    runName: string, includeLineInfo?: boolean): TimeProfile {
-  return profiler.timeProfiler.stopProfiling(runName, includeLineInfo || false);
+    runName: string, includedLineInfo?: boolean): TimeProfile {
```

Comment:
I would name the argument as `includeLineInfo` everywhere. I think there is an assumption that the same value for this argument must be passed to startProfiling and stopProfiling, so using the same name would help to express that.

Reply:
Done.